### ISSUE TITLE
Post with `list` doesn’t open — TypeError: e.get is not a function

### ISSUE THREAD

**Issue by mistermantas** [netlify/netlify-cms#1574]
**- Do you want to request a *feature* or report a *bug*?**

Bug

**- What is the current behavior?**

After logging in, everything seems to work fine, except when you try to open a post from a collection (well I only have one) that it throws a “Sorry!” error.

**- If the current behavior is a bug, please provide the steps to reproduce.**

1. Use this [config file](https://github.com/cstate/cstate/blob/master/static/admin/config.yml).
2. Source code [here](https://github.com/mistermantas/vmstatus)

![image](https://user-images.githubusercontent.com/11616378/43636106-d2c20c82-9719-11e8-9ea9-267f2fcf6ba6.png)

**- What is the expected behavior?**

That the post opens, whereas now it doesn’t.

**Comment by tech4him1** [comment]
The list widget supports two formats:
  - `field` with a single list (like you are wanting)
  - `fields` (plural) with an object for each list item

Unfortunately, single-item lists are broken right now (https://github.com/netlify/netlify-cms/issues/1139). There is a fix in progress, but it is not merged yet.

For now, you could use the alternative multi-field format. You would need to add a `name` to the system list item, and it would show up in your issue files like this:

```yaml

Affected:
  - system: tmw.media API
  - system: test
  - system: system 3
```

**Comment by mistermantas** [comment]
I can't change a core part of the frontmatter for a project nearing its last release candidate, though.

**Comment by erquhart** [comment]
Closed in #1395.
